make/nano.mk

```diff
@@ -4,25 +4,25 @@
 #
 ###########################################################
 
-NANO_SITE=http://www.nano-editor.org/dist/v2.3
-NANO_VERSION=2.3.6
-NANO_SOURCE=nano-$(NANO_VERSION).tar.gz
+NANO_SITE=http://www.nano-editor.org/dist/v3
+NANO_VERSION=3.1
+NANO_SOURCE=nano-$(NANO_VERSION).tar.xz
 NANO_DIR=nano-$(NANO_VERSION)
-NANO_UNZIP=zcat
+NANO_UNZIP=xzcat
 NANO_MAINTAINER=Mark Donszelmann <mark@donszelmann.com>
 NANO_DESCRIPTION=A pico like editor
 NANO_SECTION=editor
 NANO_PRIORITY=optional
-NANO_DEPENDS=ncurses, zlib, file
+NANO_DEPENDS=ncurses, ncursesw, zlib, file
```

Comment:
Do we need a dependency on `ncurses`? `nano` binary is linked against `ncursesw`.